### DISCUSSION TITLE
Cannot set storage charge limit for follower inverters

### DISCUSSION THREAD

**Discussion by markuswesterberg** [WillCodeForCats/solaredge-modbus-multi#702] [Q&A]
Hi! Probably something simple I'm missing but I am not able to set the Storage Charge Limit for my follower inverters 8and their batteries), I can only do so for the leader inverter. The sensors are unavailable in HA for the follower inverters. Any suggestions?

Here's what the leader inverter settings look like:
![Leader inverter](https://github.com/user-attachments/assets/e099114e-6794-46c9-a48d-e5f250cd328a)

And here's what one of my two follower inverter settings look like:
![Follower inverter](https://github.com/user-attachments/assets/f617fb94-3ba1-45a8-af85-d006ca922ee9)

As you can see, the setting sensors are unavailable.

**Comment by WillCodeForCats** [comment]
Storage control mode has to be Remote Control.

**Reply by markuswesterberg** [reply]
Perfect, works now. Thank you!!

**Comment by markuswesterberg** [comment]
Another question then. Why is it that some settings only have to be made on the leader inverter to control all other inverters/batteries, but this setting needs to be controlled on each specific inverter? Just curious.
